mods/d2k/rules/arrakis.yaml

```diff
@@ -134,17 +134,17 @@ sietch:
 	Inherits: ^Building
 	Tooltip:
 		Name: Fremen Sietch
-	Building:
+	D2kBuilding:
 		Footprint: xx xx
 		Dimensions: 2,2
 		TerrainTypes: Cliff
+		DamageTerrainTypes:
```

Comment:
Not sure if this is necessary. The tiles where it fits are all just cliff anyway.

Reply:
I had to place the sietch in the fourth atreides mission on passable ground so that the AI would attack it. Hence it would take damage in that mission.